tests/unit/lib/functions/registry.test.js

```diff
@@ -6,6 +6,7 @@ import { describe, expect, test, vi } from 'vitest'
 
 import { FunctionsRegistry } from '../../../../dist/lib/functions/registry.js'
 import { watchDebounced } from '../../../../dist/utils/command-helpers.js'
+import { getFrameworksAPIPaths } from '../../../../dist/utils/frameworks-api.js'
```

Comment:
🤔 Huh, why do all the unit tests test the built code while the integration tests test the source? That's odd.

Reply:
Good question. I'm not sure! I know we've gradually introduced vitest over ava, and there was some complexity in that transition. But that doesn't explain this. I'll take a look.